IRremoteInt.h

```diff
@@ -211,6 +211,7 @@ typedef struct {
   unsigned int timer;     // state timer, counts 50uS ticks.
   unsigned int rawbuf[RAWBUF]; // raw data
   uint8_t rawlen;         // counter of entries in rawbuf
+  uint8_t overflow;
```

Comment:
maybe. the irparams struct uses *_t types while decode_results uses the normal types. this is how it is in the Arduino library but not sure why.

According to these forum posts:
http://forum.arduino.cc/index.php?topic=42913.0
http://forum.arduino.cc/index.php?topic=139382.0

the *_t types from stdint.h used in libraries for portability and a good practice to use them. this is because bool/int/etc can be different size in different platforms.
Though this lib is esp8266 specific I agree bool should be fine. 
maybe should go over all the struct and update it in a different commit?